include/irr/asset/IAssetManager.h

```diff
@@ -171,6 +171,7 @@ namespace asset
             std::string filename = _file ? _file->getFileName().c_str() : _supposedFilename;
             io::IReadFile* file = _override->getLoadFile(_file, filename, ctx, _hierarchyLevel);
             filename = file ? file->getFileName().c_str() : _supposedFilename;
+            // TODO what should happen here (after `filename = ...;`)? Open new file if previously it was nullptr?
```

Comment:
did our discord convo resolve this question for you or not?

Reply:
resolved